src/lib/components/editor/TiptapDiffRenderer.svelte

```diff
@@ -82,11 +86,47 @@
         750
     );
 
+    $: {
+        if ($isScrollSyncEnabled && $scrollSyncSourceDiv && $scrollSyncSourceDiv != scrollSyncElement) {
+            setScroll($scrollPosition);
+        }
+    }
+
+    const setScroll = (scrollPosition: number) => {
+        if (scrollSyncElement) {
+            const scrollHeight = scrollSyncElement.scrollHeight;
+            const clientHeight = scrollSyncElement.clientHeight;
+
+            scrollSyncElement.scrollTop = scrollPosition * (scrollHeight - clientHeight);
+        }
+    };
+
+    const setScrollSyncElement = () => {
+        $scrollSyncSourceDiv = scrollSyncElement;
+    };
+
+    const handleScroll = () => {
+        if (scrollSyncElement) {
+            const scrollHeight = scrollSyncElement.scrollHeight;
+            const clientHeight = scrollSyncElement.clientHeight;
+            const scrollTop = scrollSyncElement.scrollTop;
+
+            $scrollPosition = Math.round((scrollTop / (scrollHeight - clientHeight)) * 10000) / 10000;
+        }
+    };
+
     onDestroy(() => diffWorker && diffWorker.terminate());
 </script>
 
 <div class="relative grow">
-    <div class="absolute bottom-0 left-0 right-0 top-0 overflow-y-scroll rounded-md border border-base-300 bg-white">
+    <div
+        bind:this={scrollSyncElement}
+        on:scroll={handleScroll}
+        on:mouseenter={setScrollSyncElement}
+        role="button"
```

Comment:
Could we remove the role button here and instead add:
```
                <!-- svelte-ignore a11y-no-noninteractive-element-interactions -->
```
above the div?

Reply:
That doesn't seem to have any effect on the A11y warning

Reply:
What is the a11y warning referencing? Maybe need to replace with a different rule name inside `<!-- svelte-ignore ... -->`